code/datums/wires/airlock.dm

```diff
@@ -16,9 +16,16 @@
 	add_duds(2)
 	..()
 
+/datum/wires/airlock/interact(mob/user)
+	var/obj/machinery/door/airlock/A = holder
+	if (!issilicon(user) && A.isElectrified() && A.shock(user, 100))
+		return
+
+	return ..()
+
 /datum/wires/airlock/interactable(mob/user)
 	var/obj/machinery/door/airlock/A = holder
-	if(!issilicon(user) && A.isElectrified() && A.shock(user, 100))
+	if(!issilicon(user) && A.isElectrified())
```

Comment:
Without the shock and clause here, people with insulateds will be unable to operate electrified ones.
From quick glance that shock also has power checks which might be necessary here now.

Reply:
Good catch. I can move out all the checks from `shock` into its own proc and use that for the wires.

Reply:
This has been fixed.